Dockerfile

```diff
@@ -1,89 +1,61 @@
-#############      builder       #############
+# builder
 FROM golang:1.20.6 AS builder
-
 WORKDIR /go/src/github.com/gardener/gardener
 COPY . .
-
 ARG EFFECTIVE_VERSION
-
+RUN cat Makefile
+RUN cat pkg/.import-restrictions
+RUN cat imagevector/images.yaml
```

Comment:
Why do we need this? Docker image build works without this also.

Reply:
Oops, good catch, used this for debugging :) Removed